MANIFEST.in

```diff
@@ -1,3 +1,3 @@
-recursive-include flask_template *
+recursive-include flask_create *.py
```

Comment:
This will include the entire .py files and not the directory. So it may break. Anyways I will check if this works!

Reply:
The line `recursive-include flask_template *` added no directory to the wheel.